gui/create-item.lua

```diff
@@ -230,7 +232,9 @@ function createCorpsePiece(creator, bodypart, partlayer, creatureID, casteID, ge
  local layerName
  local layerMat
  local tissueID
- if not generic then -- if we have a specified body part and layer, figure all the stuff out about that
+ local liquid = false
+ local isCorpse = bodypart < -2 -- in the hackWish function, the bodypart variable is initialized to -1, which isn't changed if the spawned item is a corpse, but it's passed to this function with two subtracted from it, making it -3 instead
```

Comment:
what's happening with the `-1 - 2` logic? what is it actually trying to do with the minus 2? this feels like it's more hacky than we need it to be, but I don't know the context

Reply:
some lists in lua start from `0`, and some seem to start from `1`. i was having an issue where the corpsepiece actually spawned was one entry further than i selected (if i pick `right upper leg`, i get `left upper leg` instead), meaning i had to adjust for that offset.

then, adding the `generic` option to the body part list bumped every entry's index forward by one, meaning i had to adjust for an offset of _two_ instead.

Reply:
i should probably put a comment of this where the offset stuff actually happens.